jerry-core/parser/js/js-parser-internal.h

```diff
@@ -273,6 +273,19 @@ typedef struct
 #endif /* PARSER_DUMP_BYTE_CODE */
 } parser_context_t;
 
+/**
+ * @}
+ * @}
+ * @}
+ */
+
+/** \addtogroup mem Memory allocation
+ * @{
+ *
+ * \addtogroup mem_parser Parser memory manager
+ * @{
+ */
+
```

Comment:
I get it. But adding these extra lines to the header does not looks nice. Is there a way to put these comments into one line?